### PR TITLE
Fix Age group bar chart for unspecified data.

### PR DESCRIPTION
**Pull request by zamuzakki**
This is PR for #1408 
**The actual data**
[cheetah.json](https://github.com/kartoza/sawps/files/13277079/cheetah.json)

**The charts**
[Kazam_screencast_00048.webm](https://github.com/kartoza/sawps/assets/7352963/f9989476-8cce-4fdc-b1e0-4fa29bf3c50b)
Population per age group for Cheetah(1)

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 12 min

File: django_project/frontend/src/containers/MainPage/Metrics/AgeGroupBarChart.tsx [hunk 1]
```diff
@@ -71,36 +74,36 @@ const AgeGroupBarChart = (props: any) => {
...
-
-    datasets.push({
-      label: 'Unspecified',
-      data: sortedData,
-      backgroundColor: 'rgba(204, 204, 204, 1)'
-    });
-  }
-
-  addUnspecifiedData();
+  //
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Metrics/AgeGroupBarChart.tsx:77]: @zamuzakki please just remove the lines if not used anymore.

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 2 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 9 min 27 sec

**Pull request merged**